src/components/ResearchGroupCard/ResearchGroupCard.jsx

```diff
@@ -1,2 +1,36 @@
 import React, { Component } from 'react';
 import './style.scss';
+import { makeCourseList } from '../../helpers'
+
+class ResearchGroupCard extends Component {
+
+	constructor(props){
+		super(props);
+		this.state={professor: props.professor, researchGroup: props.researchGroup, content: props.content, courses: props.courses};
+	}
+	
+  render () {
+    return (
+    	<div className="card">
+			<div className="card_professor">
+				<p>{this.state.professor}</p>
+			</div>
+			<div className="card_research-group">
+				<p>{this.state.researchGroup}</p>
+			</div>
+			<div className="card_content">
+				<p>{this.state.content}</p>
+			</div>
+			<hr />
+			<div className="card_courses">
+				<p>Looking For:</p>
+				<ul>
+					{makeCourseList(this.state.courses).map((course)=> <li> {course} </li>)}
```

Comment:
same as the comment I made for `ResearchGroupCardList` and the `map` property requiring the `key` prop

Reply:
Fixed now!